js/show-message.js

```diff
@@ -0,0 +1,65 @@
+import {isEscEvent} from './utils.js';
+
+const DATA_ERROR_TIME = 3000;
+const mainElement = document.querySelector('main')
+const successMessageTemplate = document.querySelector('#success').content;
+const successMessageElement = successMessageTemplate.querySelector('.success');
+const errorMessageTemplate = document.querySelector('#error').content;
+const errorMessageElement = errorMessageTemplate.querySelector('.error');
+
+// Функция показа ошибки, если не удалось получить данные с сервера:
+const showDataError = () => {
+  const errorTemplate = document.querySelector('#data-error').content;
+  const errorMessageElement = errorTemplate.querySelector('.data-error').cloneNode(true);
+  document.body.append(errorMessageElement);
+
+  setTimeout(() => {
+    errorMessageElement.remove();
+  }, DATA_ERROR_TIME);
+}
+
+// Функция закрытия сообщения об успешной отправке
+const closeSuccessMessage = () => {
```

Comment:
```suggestion
const onCloseSuccessMessage = () => {
```

_*Правила именования функций-обработчиков: https://up.htmlacademy.ru/javascript/22/criteries#d4_